### PR TITLE
ci: fix test_type arg assignment

### PR DESCRIPTION
**Pull request by karthik-us**
Set the test_type argument to the default value only if no type is specified explicitly.

### CODE REVIEW

File: mini-e2e-helm.groovy [hunk 1]
```diff
@@ -198,13 +198,14 @@ node('cico-workspace') {
 		}
 		stage('run e2e') {
 			timeout(time: 120, unit: 'MINUTES') {
-				test_type = '--test-rbd=true --test-cephfs=true --test-nfs=true'
```
Comment (nixpanic) [inline mini-e2e-helm.groovy:201]: Setting a default is much cleaner and easier to read as setting it in an `else` clause. Can you use a different variable name and not override the contents of the original?

**Review by nixpanic — CHANGES_REQUESTED** [review]

File: mini-e2e-helm.groovy (hunk 1 above)
Comment (karthik-us) [reply mini-e2e-helm.groovy:201]: Done.

**Review by nixpanic — APPROVED** [review]
Thanks!

**Review by Madhu-1 — APPROVED** [review]

**Comment by nixpanic** [comment]
@mergifyio queue

**Comment by nixpanic** [comment]
@mergifyio merge

Status updates are not reported... I don't know why :thinking: 

jjb-validate - https://jenkins-ceph-csi.apps.ocp.cloud.ci.centos.org/job/jjb-validate/303/
ci-job-validation: https://jenkins-ceph-csi.apps.ocp.cloud.ci.centos.org/job/ci-job-validation/314/

**Pull request merged**